projects/adxrs290-pmdz/Makefile

```diff
@@ -0,0 +1,10 @@
+# Uncomment to use the desired platform
+# PLATFORM = xilinx
+# PLATFORM = altera
+PLATFORM = aducm3029
```

Comment:
comment PLATFORM = aducm3029
This will enable for which platform to build  when running make.
Eg. make PLATFORM=xilinx